website/docs/r/compute_manager.html.markdown

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_compute_manager"
+description: A resource to configure a Compute Manager.
+---
+
+# nsxt_compute_manager
+
+This resource provides a method for the management of a Compute Manager.
+This resource is supported with NSX 4.1.0 onwards.
+
+## Example Usage
+
+```hcl
+resource "nsxt_compute_manager" "test" {
+  description  = "Terraform provisioned Compute Manager"
+  display_name = "test"
+  tag {
+    scope = "scope1"
+    tag   = "tag1"
+  }
+
+  server = "192.168.244.144"
+
+  credential {
+    username_password_login_credential {
+      username = "user"
+      password = "pass"
+    }
+  }
+  origin_type = "vCenter"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `access_level_for_oidc` - (Optional) Specifies access level to NSX from the compute manager.
+* `create_service_account` - (Optional) Specifies whether service account is created or not on compute manager.
+* `credential` - Login credentials for the compute manager. Should contain exactly one credential enlisted below: 
```

Comment:
This is (Required) I think?

Reply:
General question of convention: (Optional) is optional, and (Required) is required, what about those who do not have any specification of neither?